### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by jordijoangimenez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,45 @@
 <h1 align="center">5GMS Application Provider</h1>
 <p align="center">
   <img src="https://img.shields.io/badge/Status-Under_Development-yellow" alt="Under Development">
-  <img src="https://img.shields.io/github/v/tag/5G-MAG/rt-5gms-application-function?label=version" alt="Version">
```
Comment (stojkovicv) [inline README.md:4]: I actually wanted to show the Application Function version here, since the management UI is dependent from it. Maybe adding both...?

**Comment by jordijoangimenez** [comment]
For the moment let's put this PR on hold as it's outdated.

@dsilhavy, we agreed the CLI, Management UI, QoE UI READMEs should be moved to the Getting-Started pages repo, right?

If that's the case I'll do it and check the cross-links.

File: README.md (hunk 1 above)
Comment (stojkovicv) [reply README.md:4]: Management UI supports all M1 operations for Application Function **v1.4.1** or later. However, seems more reasonable to add this information into the Readme, and to replace AF badge with AP version.
So this change is good.

Comment (jordijoangimenez) [reply README.md:4]: @stojkovicv Oki! If you want keep updating the documentation branch, I won't work on it for the time being. Once we get confirmation from Daniel, I will sync my documentation branch, make further changes and resend the PR if needed.

**Comment by dsilhavy** [comment]
> For the moment let's put this PR on hold as it's outdated.
> 
> @dsilhavy, we agreed the CLI, Management UI, QoE UI READMEs should be moved to the Getting-Started pages repo, right?
> 
> If that's the case I'll do it and check the cross-links.

Yes exactly, let's add all information on usage to the Getting-Started repo and only keep the installation and build instructions in the dedicated repos

**Comment by jordijoangimenez** [comment]
For the moment, I have updated the links that were referring to the old wikis. The links now point to the Application Function usage under Getting-Started.

**Review by dsilhavy — APPROVED** [review]

**Pull request merged**